TYPO3.Media/Classes/TYPO3/Media/Domain/Service/ImageService.php

```diff
@@ -192,7 +192,7 @@ protected function getOptionsMergedWithDefaults(array $additionalOptions = array
         $quality = isset($defaultOptions['quality']) ? (integer)$defaultOptions['quality'] : 90;
         if ($quality < 0 || $quality > 100) {
             throw new InvalidConfigurationException(
-                sprintf('Setting "TYPO3.Media.image.defaultOptions.quality" allow only value between 0 and 100, current value: %s', $quality),
+                sprintf('Setting "Neos.Media.image.defaultOptions.quality" allow only value between 0 and 100, current value: %s', $quality),
```

Comment:
The help message will point to a non-existing setting (for now), so don't merge without actual vendor name change.